cardano-cli/test/cardano-cli-golden/Test/Golden/Shelley/TextEnvelope/Tx/Tx.hs

```diff
@@ -47,3 +47,40 @@ hprop_golden_shelleyTx = propertyOnce . H.moduleWorkspace "tmp" $ \tempDir -> do
   -- Check the newly created files have not deviated from the
   -- golden files
   checkTxCddlFormat referenceTx transactionFile
+
+
+hprop_golden_checkIfConstitutionalCommitteeKeyCanSign :: Property
+hprop_golden_checkIfConstitutionalCommitteeKeyCanSign = propertyOnce . H.moduleWorkspace "tmp" $ \tempDir -> do
+  -- Reference keys
+  let referenceTx = "test/cardano-cli-golden/files/golden/conway/tx"
```

Comment:
Can you put this in `test/cardano-cli-golden/files/input/conway/tx` instead?

I'd like to get to the point where only golden files are in the `golden` directory.